querybook/server/lib/table_upload/exporter/s3_exporter.py

```diff
@@ -32,13 +32,21 @@
         if true, the upload root path is inferred by locationUri specified in hms
         to use this option, the engine must be connected to a metastore that uses
         HMSMetastoreLoader (or its derived class)
+        if false, it will be created as managed table, whose location will be determined automatically by the query engine.
     - table_properties (List[str]): list of table properties passed, this must be query engine specific.
         Checkout here for examples in SparkSQL: https://spark.apache.org/docs/latest/sql-ref-syntax-ddl-create-table-hiveformat.html#examples
         For Trino/Presto, it would be the WITH statement: https://trino.io/docs/current/sql/create-table.html
 """
 
 
 class S3BaseExporter(BaseTableUploadExporter):
+    def __init__(self, exporter_config: dict = {}):
+        if ("s3_path" not in exporter_config) and (
```

Comment:
actually, we don't need use_schema_location to be provided, i guess neither needs to supplied and the default
would just be using the table location